atlp-rwanda/c4-barefoot-frontend
# Ft assign users to managers

# Description
This PR is for a feature implementing to assign users to respective managers.
Once testing this feature, a list of verified users should be presented and on each user, a list of managers in a selection should be presented. 
A client can choose a manager respective to a users to be assigned to. Changes are saved by clicking on the `SAVE`.

Every subfeature should be working respective to the [mockups](https://www.figma.com/file/hM1DgjHISuEDVenacmEyo7/Barefoot-Nomad?node-id=2%3A704)
## Type of change
- FEATURE
Please check the options that are relevant.


# How should this be manually tested or reproduced?
Once you run the app, ensure you are logged in with a manager, then open the route `/assign-users-to-managers/`
- Clone this repo
- Run the command `npm i`
- Run the command `npm start`

# Checklist:


# Screenshots (if applicable)
